medical/models/medical_patient.py

```diff
@@ -1,105 +1,117 @@
-#
-# Tech-Receptives Solutions Pvt. Ltd.
-# Copyright (C) 2004-TODAY Tech-Receptives(<http://www.techreceptives.com>)
-# Special Credit and Thanks to Thymbra Latinoamericana S.A.
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as
-# published by the Free Software Foundation, either version 3 of the
-# License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-# #############################################################################
-from openerp import models, fields, api
+# Copyright 2004 Tech-Receptives
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
 from datetime import datetime
-from openerp.tools.translate import _
 from dateutil.relativedelta import relativedelta
 
 
 class MedicalPatient(models.Model):
-    '''
+    """
     The concept of Patient included in medical.
-    '''
+    """
     _name = 'medical.patient'
     _description = 'Medical Patient'
-    _inherits = {'res.partner': 'partner_id', }
+    _inherits = {'res.partner': 'partner_id'}
 
-    @api.one
+    age = fields.Char(
+        compute='_compute_age',
+    )
+    identification_code = fields.Char(
+        string='Internal Identification',
+        help='Patient Identifier provided by the Health Center.'
+             '(different from the Social Security Number)',
+    )
+    general_info = fields.Text(
+        string='General Information',
+    )
+    active = fields.Boolean(
+        default=True,
+    )
+    is_deceased = fields.Boolean(
+        compute='_compute_is_deceased',
+    )
+    partner_id = fields.Many2one(
+        string='Related Partner',
+        comodel_name='res.partner',
+        required=True,
+        ondelete='cascade',
+        index=True,
+    )
+    marital_status = fields.Selection([
+        ('s', 'Single'),
+        ('m', 'Married'),
+        ('w', 'Widowed'),
+        ('d', 'Divorced'),
+        ('x', 'Separated'),
+        ('z', 'law marriage'),
+    ], )
+    is_pregnant = fields.Boolean(
+        help='Check this if the patient if pregnant',
+    )
+    date_death = fields.Datetime(
+        string='Deceased Date',
+    )
+
+    @api.multi
     def _compute_age(self):
         """
-        age computed depending of the birth date of the
+        Age computed depending of the birth date of the
         membership request
         """
         now = datetime.now()
```

Comment:
Typically yes, but this is being used in the `relativedelta` a bit below this so the object was needed in this instance. I could have `fields.Datetime.from_string(fields.Datetime.now())`, but that would have been kind of clunky IMO